core/templates/dev/head/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -95,7 +98,20 @@ oppia.factory('ExplorationEngineService', [
     // Param changes to be used ONLY in editor preview mode.
     var manualParamChanges = null;
     var initStateName = null;
-    var version = GLOBALS.explorationVersion;
+    var version = UrlService.getExplorationVersionFromUrl();
+    if (version) {
+      ReadOnlyExplorationBackendApiService
+        .loadExploration(_explorationId, version)
+        .then(function (exploration) {
+          version = exploration.version;
+        });
+    } else {
+      ReadOnlyExplorationBackendApiService
+        .loadExploration(_explorationId, version)
+        .then(function (exploration) {
+          version = exploration.version;
+        });
+    }
```

Comment:
Why are these two the same?

Reply:
Changed.
The second was supposed to load the exploration without the version (latest exploration).